### PR TITLE
feat: Upload Portable PDB files using the Module Version Id of the corresponding PE

### PR DESCRIPTION
**Pull request by Swatinem**
Does what the title says. Though not sure this is the best solution to this really.

### CODE REVIEW

File: src/utils/dif_upload.rs [hunk 1]
```diff
@@ -1201,6 +1254,57 @@ fn create_il2cpp_mappings<'a>(difs: &[DifMatch<'a>]) -> Result<Vec<DifMatch<'a>>
     Ok(line_mappings)
 }
 
+fn resolve_ppdb(mut difs: Vec<DifMatch>) -> Vec<DifMatch> {
```
Comment (loewenheim) [inline src/utils/dif_upload.rs:1257]: I think this function might as well directly mutate its argument.
```suggestion
fn resolve_ppdb(difs: &mut Vec<DifMatch>) {
```

File: src/utils/dif_upload.rs [hunk 2]
```diff
@@ -77,6 +78,8 @@ impl<'a> Iterator for DifChunks<'a> {
 /// [`DifMatch`] instead of this instead.
 enum ParsedDif<'a> {
     Object(Box<Object<'a>>),
+    // TODO: consider making this an `Object` in symbolic-debuginfo?
```
Comment (loewenheim) [inline src/utils/dif_upload.rs:81]: I don't think that's a good idea. PortablePdb has almost none of the features one would expect from an `Object`.

**Comment by mattjohnsonpint** [comment]
@Swatinem - is this still needed? I think the latest is working without it, no?

**Comment by Swatinem** [comment]
Indeed, this is obsolete now.

**Pull request closed without merging**